components/generic/ContentCard.vue

```diff
@@ -157,10 +110,15 @@
     }
 
     .card-link {
+      min-height: 100%;
       color: $black;
       &:hover {
         color: $black;
       }
     }
+
+    .view-more {
+      float: right;
+    }
```

Comment:
Let's keep it to the left, I think that is more consistent with the proposed design.